test-requirements.txt

```diff
@@ -9,3 +9,4 @@ testscenarios>=0.5.0 # Apache-2.0/BSD
 testtools>=2.3.0 # MIT
 beautifulsoup4>=4.8.0 # MIT
 pylint==1.9.4 # GPLv2
+pandas==1.2.4
```

Comment:
This isn't necessary as the python example is not actually compiled or run. We only run the AST parser on it to detect security issues.